pkg/cmd/roachtest/spec/cluster_spec.go

```diff
@@ -526,8 +526,8 @@ func (s *ClusterSpec) Expiration() time.Time {
 
 // TotalCPUs is the total amount of CPUs allocated for a cluster.
 func (s *ClusterSpec) TotalCPUs() int {
-	if !s.WorkloadNode {
+	if s.WorkloadNodes == 0 {
```

Comment:
Nit: can simplify by removing since L532 does everything now; also it might be faster since simple arithmetic doesn't take a lot of cpu cycles whereas a branch mis-prediction, and the resulting pipeline stall could end up being more.

Reply:
Ah good catch, removed!